_posts/en/newsletters/2021-01-27-newsletter.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Bitcoin Optech Newsletter #133'
+permalink: /en/newsletters/2021/01/27/
+name: 2021-01-27-newsletter
+slug: 2021-01-27-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a meeting to discuss taproot activation
+mechanisms and includes our regular sections with top questions and
+answers from the Bitcoin StackExchange, a list of releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Scheduled meeting to discuss taproot activation:** Michael Folkson
+  [announced][folkson announce] that a meeting will be held at <time
+  datetime="2021-02-02 19:00-0000">19:00 UTC on February 2nd</time> in
+  the [##taproot-activation][] IRC channel on irc.freenode.net to
```

Comment:
irc.freenode.net looks like it wants to be a link but isn't, not sure here but perhaps add a link or use a less url-looking name like freenode